tests/integration/test_dicom_import.py

```diff
@@ -6,9 +6,25 @@
 from dicom_image_tools.dicom_handlers.dicom_import import import_dicom_from_folder, import_dicom_file
 from dicom_image_tools.dicom_handlers.dicom_study import DicomStudy
 
+def test_import_dicom_from_file_should_parse_kv_ma_ms_tags():
+
+    file = Path(__file__).parent.parent / 'test_data' / 'io' / 'iotest_60kv_7ma_20ms.dcm'
+    
+    expected_output = dict(kV=60.0, mA=7.0, ms=20.0)
```

Comment:
maybe turn this into three variable instead (`expected_kv`, `expected_ma`, and `expected_ms`) and use 3 assert statements below instead? That might make it easier to quickly find which, if any of these fail.

Reply:
great idea, fixed